src/Simulations/run.jl

```diff
@@ -131,7 +140,12 @@ function time_step!(sim::Simulation)
 
     else # business as usual...
         Δt = aligned_time_step(sim, sim.Δt)
-        time_step!(sim.model, Δt, callbacks=model_callbacks)
+        if Δt > (sim.Δt / 1e10)
```

Comment:
Seeing as this is a `Simulation` thing, we could add this as a property to `Simulation` with a default 0 (or `nothing` if we want...)

```julia
Simulation.minimum_relative_step = 0
```

I would write the logic as

```julia
if Δt < sim.minimum_relative_step * sim.Δt
    next_time = next_actuation_time(sim)
    @warn "Reseting clock to $next_time and skipping aligned time step Δt = ", Δt)
    sim.model.clock.time = next_time
else
    time_step!(sim.model, Δt, callbacks=model_callbacks)
end
```

This makes is clear that the information to pay attention to is when the time-step is too small I think.

Reply:
Assuming the logic also works with `next_actuation_time(sim)`, I agree with this. Is this the way we want to solve this issue though? I think in the past we were considering skipping a step a rather "hacky" solution.

Reply:
I do think its cleaner to use a digitized / discretized time variable with a minimum increment and @Sbozzolo has actually been developing such a thing. But also if you _need_ this then we should consider it. Although, nobody else is complaining (but I guess people are shy)

Reply:
For the record, I do know of at least two people that have reported this issue to me through personal communication, but never reported it online.

Reply:
ok